src/include/sof/audio/kp_common.h

```diff
@@ -0,0 +1,6 @@
+#include <stdint.h>
+
+struct kp_data {
```

Comment:
Anything that is part of kpbuf public interface (internal to other fw components/units) may go to kpb.h or kpb_public.h (the latter follows current organization where the <com>.h headers declare rather private data).

Reply:
kp_common.h, or kpb_public.h as you suggested has the same intention - provide a public/common data structure to be shared by key phrase related components.